arangod/IResearch/IResearchLink.cpp

```diff
@@ -1038,10 +1073,10 @@ Result IResearchLink::init(
   bool const sorted = !meta._sort.empty();
   auto const& storedValuesColumns = meta._storedValues.columns();
   TRI_ASSERT(meta._sortCompression);
-  auto const primarySortCompression = meta._sortCompression
-      ? meta._sortCompression
-      : getDefaultCompression();
-  if (ServerState::instance()->isCoordinator()) { // coordinator link
+  auto const primarySortCompression =
+      meta._sortCompression ? meta._sortCompression : getDefaultCompression();
+  bool clusterWideLink{true};
```

Comment:
let's move it under `} else if (ServerState::instance()->isDBServer()) {`, as it only makes sense there

Reply:
I can't I need this variable in another scope